app/src/components/market/market_home.tsx

```diff
@@ -62,7 +62,7 @@ export const MarketHome: React.FC<Props> = (props: Props) => {
         {RemoteData.hasData(markets) &&
           markets.data.length > 0 &&
           markets.data.slice(0, count).map(item => {
-            return <ListItem key={item.conditionId} data={item}></ListItem>
+            return <ListItem key={`${item.address}_${item.conditionId}`} data={item}></ListItem>
```

Comment:
This should be fine but I'm curious, how did you get two outcomes with the same `conditionId` (assuming that's why you did this).

Reply:
Mmmm this is happening with rinkeby, you can check the screenshot, Why is this happening ? I think we need to investigate more deeply...

![Screenshot_20191219_103141](https://user-images.githubusercontent.com/1144028/71177509-f580ba80-224a-11ea-9ee9-37a0cc1d9b38.png)

Reply:
Weird. Well, it's OK then.